### PR TITLE
bots: Move fedora-testing image to fedora-29

### PR DESCRIPTION
**Pull request by KKoukiou**
* [x] image-refresh fedora-testing

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-bskpf.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10557-20181112-150505/

File: bots/images/scripts/fedora-28.bootstrap [hunk 1]
```diff
@@ -18,4 +18,4 @@
 # 02110-1301 USA.
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "http://dl.fedoraproject.org/pub/fedora/linux/releases/28/Server/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "http://dl.fedoraproject.org/pub/fedora/linux/releases/29/Server/x86_64/os/"
```
Comment (martinpitt) [inline bots/images/scripts/fedora-28.bootstrap]: Don't change the fedora-28.bootstrap script, just change the fedora-testing.bootstrap symlink to point to fedora-29.bootstrap. :-)

**Comment by martinpitt** [comment]
Aborting image refresh due to above review; to restart, just re-push and drop the FAIL marker.

File: bots/images/scripts/fedora-28.bootstrap (hunk 1 above)
Comment (KKoukiou) [reply bots/images/scripts/fedora-28.bootstrap]: ups, good catch.

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-testing-20181112-160000

**Comment by cockpituous** [comment]
image-refresh in progress on bos-tasks-sl5c2.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-10557-20181112-160404/

**Comment by cockpituous** [comment]
image-refresh fedora-testing done: https://github.com/cockpituous/cockpit/commits/image-refresh-fedora-testing-20181112-165858

**Comment by martinpitt** [comment]
Yep, these Fedora regressions are real. :-(

**Pull request merged**